ios/Handlers/RNNativeViewHandler.m

```diff
@@ -99,21 +99,21 @@ - (void)handleTouchDown:(UIView *)sender forEvent:(UIEvent *)event
 
     [self sendEventsInState:RNGestureHandlerStateActive
              forViewWithTag:sender.reactTag
-              withExtraData:[RNGestureHandlerEventExtraData forPointerInside:YES]];
+              withExtraData:[RNGestureHandlerEventExtraData forPointerInside:YES withNumberOfTouches:_recognizer.numberOfTouches]];
```

Comment:
I believe numberOfTouches should be extracted from `event` not `_recognizer` in this case

Reply:
Or just stop sending numberOfPoiners for UIControl subclasses as it may not be that important